ArmoryQt.py

```diff
@@ -5230,21 +5267,35 @@ def unregisterProgressCallback(self, id):
    app_dir = "./"
    try:
       app_dir = os.path.dirname(os.path.realpath(__file__))
-   except:
-      if OS_WINDOWS and getattr(sys, 'frozen', False):
-         app_dir = os.path.dirname(sys.executable)
+   except Exception:
+      try:
+         # On frozen builds (Windows), sys.frozen is set to True
+         if OS_WINDOWS and sys.frozen:
+            app_dir = os.path.dirname(sys.executable)
+      except Exception:
+         pass
    translator.load(TheSettings.getGuiLanguage(), os.path.join(app_dir, "lang/"))
    QAPP.installTranslator(translator)
 
-   #setup main dialog
+   # 3) Create main window after setup manager is closed
    armoryMainWindow = ArmoryMainWindow(splashScreen=SPLASH)
 
-   #start cppbridge
-   TheBDM.startBridge(getBridgeArgList(), armoryMainWindow.networkReadyCallback)
+   # Bridge already started; ensure the main window receives the ready callback
+   try:
```

Comment:
what does this block do? spawn armoryqt from the setup manager on setup completion instead